src/it/java/com/google/checkstyle/test/base/AbstractIndentationTestSupport.java

```diff
@@ -60,6 +60,18 @@
         return getLinesWithWarnAndCheckComments(fileName, TAB_WIDTH);
     }
 
+    /**
+     * Returns line numbers for lines with Warn comments.
+     * This implementation uses {@link #getIndentFromComment(String)},
```

Comment:
It is better to avoid such sentence, no such details are required.
Please remove from other javadoc that add.

Reply:
Done! Hope the changes are fine.